### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by chambliss**
Various small changes based on discussion & feedback from prior PRs, and a few deletions of files that are no longer needed. No major changes to the repo's functionality - the biggest change was revamping the monolingual demo to be cleaner.

Also added instructions and environment file for creating a virtual environment with conda.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [flake8]
+# Exceptions:
```
Comment (ninalopatina) [inline .flake8]: thanks for adding this in!

File: README.md [hunk 2]
```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
...
 ## Installation
 
-(will make available via `pip install` and `conda install` when project is closer to completion - hit the Watch button if you'd like to be updated on progress!)
+**NOTE**: Since there are quite a few requirements, creating a virtual environment for this repo is strongly recommended.
+
+**Instructions**: Use `conda` to create the environment. Currently environment creation via pip is not supported.
+
+To create the environment with `conda`, all you need to do is navigate to the `Multilingual_NER` directory and run the following in the command line:
+
+`conda env create -f environment.yml`
```
Comment (ninalopatina) [inline README.md]: great addition. This works well. However, when I run the streamlit demo, it is missing the xx_ent_wiki_sm, so python -m spacy download xx_ent_wiki_sm should go somewhere in your install instructions (inside the environment)

File: README.md [hunk 3]
```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
...
--   **NOTE**: Since there are quite a few requirements, we recommend creating a virtual environment when using this package. (use `requirements.txt` when creating environment from scratch)
--   pytorch-transformers 1.1.0+, torch 1.2.0+
--   spaCy (2.1.0+)
--   TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
--   numpy 1.15.4+, pandas 0.25.0+
--   pre-commit 1.18.0+
--   streamlit 0.4.0+
--   (Deployment dependencies - TBD)
--   Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
+Once you have a usable model checkpoint, open `python/demo.py` and set the path to the model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py en`, which opens a web app you can use to test the model's predictions interactively. For the Russian demo, change `en` to `ru`.
```
Comment (ninalopatina) [inline README.md]: the path instructions here are no longer relevant based on how you have now set this up (which is great, btw). The demo works great!

File: README.md [hunk 4]
```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
...
 
 ## Models
 
--   The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
--   The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
+- The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
+- The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
+- Before running the demo or either of the spaCy evaluation notebooks, be sure to run `python -m spacy download MODEL_NAME`.
```
Comment (ninalopatina) [inline README.md]: I see the instructions here for the spacy download; these should go higher up, before the demo running instructions

File: python/demo.py [hunk 5]
```diff
@@ -92,57 +50,55 @@ def predict_on_text(input_text):
...
-            + "\N{DISAPPOINTED FACE} Try a longer string."
-        )
-        plot_title = "Number of words/word-pieces matching each label:"
-    else:
-        user_prompt = "Какой текст вы хотите предсказать?"
-        found_msg = "Модель нашла эти сущности:"
-        not_found_msg = "Объекты не найдены. Попробуйте длинное предложение."
-        plot_title = "Количество слов / частей слова, соответствующих каждому ярлыку:"
+    # Create selectbox for users to select checkpoint
+    CHK_PATH = st.selectbox("Model checkpoint:", tuple(available_chkpts))
```
Comment (ninalopatina) [inline python/demo.py]: great addition!

File: README.md (hunk 3 above)
Comment (chambliss) [reply README.md]: Very true, I'll amend this! And thanks! :)

**Pull request merged**